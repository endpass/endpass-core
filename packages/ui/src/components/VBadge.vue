<template>
  <span class="badge"><slot /></span>
</template>

<script>
export default {
  name: 'VBadge',
};
</script>

<style lang="scss">
.badge {
  display: inline-block;
  padding: 0.25em 0.4em;
  font-size: 75%;
  font-weight: 700;
  line-height: 1;
  text-align: center;
  white-space: nowrap;
  vertical-align: baseline;
  border-radius: 0.25rem;
}
</style>
