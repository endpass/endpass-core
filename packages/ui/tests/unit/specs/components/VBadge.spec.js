import { shallowMount } from '@vue/test-utils';
import VBadge from '@/components/VBadge';

describe('VBadge', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(VBadge);
  });

  it('should be a correct VBadge', () => {
    expect(wrapper.name()).toBe('VBadge');
    expect(wrapper.isVueInstance()).toBeTruthy();
    expect(wrapper.element).toMatchSnapshot();
  });
});
