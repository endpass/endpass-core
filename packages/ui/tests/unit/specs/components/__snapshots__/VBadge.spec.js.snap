// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VBadge should be a correct VBadge 1`] = `
<span
  class="badge"
/>
`;
